lusidtools/cocoon/cocoon.py

```diff
@@ -92,12 +96,13 @@ def get_alphabetically_first_identifier_key(
             return f"{first_unique_identifier_alphabetically}: {instrument.identifiers[first_unique_identifier_alphabetically].value}"
 
         return api_factory.build(lusid.api.InstrumentsApi).upsert_instruments(
+            scope=kwargs["scope"],
```

Comment:
This seems to be the only change in this file (which is fine). The rest of the changes are formatting ones which might be best to leave because they don't all make sense.

Reply:
use `.get` in case `scope` is not in the dict